test.js

```diff
@@ -1,25 +1,4009 @@
-import {describe, it} from 'mocha';
-import * as chai from 'chai';
-import {expect} from 'chai';
+import {beforeEach, describe, it} from 'mocha';
 import parser from './build/index.js';
 
-chai.should();
+import * as chai from 'chai';
+chai.should()
 
+// const { expect } = require('chai')
 // const fs = require('fs')
+// const parser = require('../build/index.cjs')
 // const path = require('path')
 
 describe('yargs-parser', function () {
-  // it('should parse a "short boolean"', function () {
-  //  const parse = parser(['-b'])
-  //  parse.should.not.have.property('--')
-  //  parse.should.have.property('b').to.be.ok.and.be.a('boolean')
-  //  parse.should.have.property('_').with.length(0)
-  //})
+  /*it('should parse a "short boolean"', function () {
+    const parse = parser(['-b'])
+    parse.should.not.have.property('--')
+    parse.should.have.property('b').to.be.ok.and.be.a('boolean')
+    parse.should.have.property('_').with.length(0)
+  })*/
 
   it('should parse a "long boolean"', function () {
-    const parse = parser(['--bool'])
+    const parse = parser('--bool')
     parse.should.not.have.property('--')
     parse.should.have.property('bool', true)
     parse.should.have.property('_').with.length(0)
   })
+
+  
+  it('should place bare options in the _ array', function () {
+    const parse = parser('foo bar baz')
+    parse.should.have.property('_').and.deep.equal(['foo', 'bar', 'baz'])
+  })
+
+  // Proposing that '-' is always short opts.
+  /*it('should set the value of the final option in a group to the next supplied value', function () {
```

Comment:
Once we implement short options in https://github.com/pkgjs/parseargs/pull/50 this test will mostly pass, the difference is that `s` would not be set to `meow` with the simplification we're proposing.

Reply:
We ended up opting for the behavior documented in this test:

https://github.com/pkgjs/parseargs/pull/59